### PR TITLE
tony's exercise

### PR DESCRIPTION
**Pull request by tonytanaka**
please review my code

### CODE REVIEW

File: tonytanaka/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Question 1
+// NaN stands for Not-A-Number
+let ex1a = "hello" * 1;
+console.log(typeof ex1a);
```
Comment (ciamac-da) [inline tonytanaka/index.js]: In line 4 you have to check if  ex1a is NaN or not! And result should be true or false!
console.log(isNaN(ex1a));

File: tonytanaka/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+let ex1b = "100em";
+console.log(isNaN(ex1b));
```
Comment (ciamac-da) [inline tonytanaka/index.js]: Also here in line 6 you have to check if ex1b is NaN or not! And result should be true or false!
console.log(isNaN(ex1b));

**Review by ciamac-da — CHANGES_REQUESTED** [review]

File: tonytanaka/index.js (hunk 1 above)
Comment (tonytanaka) [reply tonytanaka/index.js]: ok I will incude it

File: tonytanaka/index.js (hunk 2 above)
Comment (tonytanaka) [reply tonytanaka/index.js]: this line already exists

Comment (ciamac-da) [reply tonytanaka/index.js]: Oh indeed, my bad!

**Review by ciamac-da — APPROVED** [review]
👍🏻

**Pull request merged**